crates/smashline/src/lib.rs

```diff
@@ -47,6 +47,13 @@ impl std::fmt::Display for Priority {
     }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone, PartialEq, PartialOrd)]
+pub struct Costume {
+    pub min: i32,
```

Comment:
Instead of a hardcoded range, please consider using a Vec<usize>.

```rs
pub struct Costume {
    pub vec: Vec<usize>
}

pub impl Costume {
    pub fn new(vec: Vec<usize>) -> Costume {
        Costume { vec }
    }
}

agent.set_costume([8, 9, 10, 12, 13, 14, 15].to_vec());
```

It's a little more verbose but it's much more clear on what it's doing. This also gives the freedom of skipping costumes, like if you wanted to install scripts only on Radiant Dawn Ike.